### ISSUE TITLE
Opinions on backwards-incompatible changes to fused-effects?

### ISSUE THREAD

**Issue by robrix** [aurapm/aura#547]
Hello! I’m the primary author of `fused-effects`, and I was delighted to learn that `aura` uses our package 😊

We’re considering some backwards-incompatible organizational changes to the package, but I’m not keen on doing that without input from users on how this will affect them. To that end, I’ve opened a PR as an RFC, and would love to hear your thoughts on this if and when you have a moment:

https://github.com/fused-effects/fused-effects/pull/204

**TL;DR:** We’d like to have users import e.g. `Control.Carrier.State.Strict` instead of `Control.Effect.State`; would that sort of change be acceptable in your view?

Thanks for using `fused-effects`, and please let me know if there’s anything we can do to make the library more useful for you! 💖

**Comment by fosskers** [comment]
@robrix oh sure, that's a super trivial change. Break away!
